build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         maven { url "https://repo.spring.io/milestone" }
     }
     dependencies {
-        classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.0.0.M2'
+        classpath 'org.springframework.boot:spring-boot-gradle-plugin:1.5.2.RELEASE'
```

Comment:
2.0.0.M2 broke ace-editor for some reason. It's currently broken on master branch (Travis couldn't notice it). Furthermore we didn't find any good reason why we switched spring-boot version in the earlier merge.